phpunit.xml.dist

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- TODO set checkForUnintentionallyCoveredCode="true" once https://www.drupal.org/node/2626832 is resolved. -->
+<!-- PHPUnit expects functional tests to be run with either a privileged user
+ or your current system user. See core/tests/README.md and
+ https://www.drupal.org/node/2116263 for details.
+-->
+<phpunit bootstrap="vender/weitzman/drupal-test-traits/src/bootstrap.php" colors="true"
+         beStrictAboutTestsThatDoNotTestAnything="true"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutChangesToGlobalState="true"
+         printerClass="\Drupal\Tests\Listeners\HtmlOutputPrinter">
+  <php>
+    <!-- Set error reporting to E_ALL. -->
+    <ini name="error_reporting" value="32767"/>
+    <!-- Do not limit the amount of memory tests take to run. -->
+    <ini name="memory_limit" value="-1"/>
+    <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
+    <env name="SIMPLETEST_BASE_URL" value=""/>
```

Comment:
I have tried both `http://localhost` and `http://nginx`, neither works. 
my question is that , what is correct value for this ,  is it possible to access `base url` from the container.

Reply:
The value should be whatever https://github.com/dpc-sdp/dev-tools/blob/b6b250005595167288e7908f2792d6090eabfd1c/.env#L35 is set to.

Reply:
Thanks @GROwen !  yeah , we can access it from the outside of the container, but what if I would like to access it from containers like `php` or `cli` container?

Reply:
I think , I just missed the port number.